### PR TITLE
Improve layout and labelling of the workshops

### LINKED ISSUE #851: Improve layout and labelling of the workshops section on the individual chapter pages
## Description of the issue 📄
Add Upcoming events label to upcoming events section. Add better visual separation between upcoming and past events sections and between events in a specific section as well.

## Screenshots 📷
Current layout
screen shot 2018-10-17 at 1 35 43 pm

## Steps to fix 🛠
Update relevant templates and CSS files.

## To do 📋
* [ ] Claim this issue (comment below, or assign yourself if you are part of the codebar org)
* [ ] Fork and clone the repository
* [ ] Update the relevant files. Follow the steps to fix section in this issue.
* [ ] Commit your changes as one commit. Use the title of this issue as your commit message
* [ ] Submit a pull request
* [ ] Mention this issue in the PR description by including it's number
* [ ] Have your pull request reviewed & merged by a codebar team member

### PR DESCRIPTION
**Pull request by stphnnnn**
I've made some improvements to the labelling of workshops and workshop card designs.
See discussion on the issue here: #851 .

Below are screenshots of the latest changes...

Chapter page:
screenshot 2019-02-21 at 00 00 46

Home page:
screenshot 2019-02-20 at 23 58 56

This is a real pull request as the draft didn't seem to go anywhere.

Open to feedback as I may have made some controversial changes 😆

Closes #851

### CODE REVIEW

**Comment by KimberleyCook** [comment]
I really like these imporvement thank you @stphnnnn. 

Out of curiosity how would the logos look like if an "Event" had a host and 4 sponsors? This does happen with 2 events we run throughout the year, uncodbar and 24PR.

**Comment by stphnnnn** [comment]
@KimberleyCook 
Glad you like the changes and good question.

I've made a few changes and here's how 4 logos looks at two different break points:

screenshot 2019-03-02 at 17 56 00

screenshot 2019-03-02 at 17 56 12

**Comment by matyikriszta** [comment]
@stphnnnn great changes so far! I think it would be better if we made the chapter the workshop/event belongs to even more prominent and space the cards out a little bit more so they are more distinct. Out of curiosity could you paste a picture of how this layout would look on a gray background and on a white background with drop shadows on each of the cards? My concern is that the cards still don't stand out enough from each other and the bg. And thirdly I think the date text the cards are grouped by could be bigger.

File: app/views/chapter/show.html.haml [hunk 1]
```diff
@@ -24,20 +24,18 @@
   .row
     .large-8.columns
       - if @upcoming_workshops.any?
-        - @upcoming_workshops.each do |date, workshops|
-          .row.grouped-events
-            .medium-12.columns
-              %h4= date
-            = render workshops
+        .events-list
+          %h3 Upcoming Events
```
Comment (matyikriszta) [inline app/views/chapter/show.html.haml]: Any reason why this is not a H2?

**Comment by stphnnnn** [comment]
Thanks for the feedback @matyikriszta, here are two screenshots with the drop shadows on white and grey:

screenshot 2019-03-06 at 18 59 49

screenshot 2019-03-06 at 19 08 23

I've bumped up the size of the date headings but I think it's also worth adding a "Upcoming Events" to the homepage to make it clear to users what they're looking at.

**Comment by matyikriszta** [comment]
@stphnnnn ok, great, I do like them with the drop shadow, can we maybe just make them a little bit less prominent? Otherwise this is starting to come together nicely.

**Comment by stphnnnn** [comment]
Yup, I agree. Have a look at the below (I'll push these changes now)

Screenshot 2019-03-09 at 15 26 40

**Comment by KimberleyCook** [comment]
I think this looks good thanks for your work on this @stphnnnn.  @matyikriszta I'm happy for this to get merged in if you are?

**Comment by matyikriszta** [comment]
@KimberleyCook yes, I'm happy with this as well. Thanks @stphnnnn for you work on this and for working with us through the feedback.

**Comment by matyikriszta** [comment]
@stphnnnn there are 3 tests that need updating after the changes (they rely on certain classes being on the page and these got removed I think). Do you think you could update these so the test suite passes? You can see the failing tests by clicking on the Details button next to the continuous-integration/travis-ci/pr above.

**Comment by stphnnnn** [comment]
@matyikriszta of course! I'll sort those sorted later today 💯

**Comment by stphnnnn** [comment]
@KimberleyCook @matyikriszta 
Sorry for the delay with this, looks like everything's passing now 👍

**Comment by KimberleyCook** [comment]
@stphnnnn a deploy was done today, so these changes are now in production. It's looking good 💃

**Comment by stphnnnn** [comment]
Woo! 🎈

**Comment by alextercete** [comment]
Just thought I'd come by and say I love the new look. Well done @stphnnnn! 👍

**Pull request merged**